gapis/gfxapi/gles/resources.go

```diff
@@ -108,7 +106,7 @@ func (t *Texture) ResourceData(ctx context.Context, s *gfxapi.State) (interface{
 		return &gfxapi.Cubemap{Levels: levels}, nil
 
 	default:
-		return nil, &service.ErrDataUnavailable{Reason: messages.ErrNoTextureData(t.ResourceHandle())}
+		return nil, fmt.Errorf("Unhandled texture type: %v", t.Kind)
```

Comment:
The old message handled the case that the name had been generated, but nothing had been bound yet. Please reintroduce the old error into a new `case 0:`